chainercv/utils/testing/assertions/assert_is_point_dataset.py

```diff
@@ -23,9 +23,9 @@ def assert_is_point_dataset(dataset, n_point=None, n_example=None,
             If this argument is specified, this function picks
             examples ramdomly and checks them. Otherwise,
             this function checks all examples.
-        no_mask (bool): If :obj:`True`, we assume that
-            point mask is always not contained.
-            If :obj:`False`, point mask may or may not be contained.
+        no_visible (bool): If :obj:`True`, we assume that
+            visibility mask is always not contained.
+            If :obj:`False`, point visible may or may not be contained.
```

Comment:
`visibility mask`?. I think we can use `:obj:visible` instead of "visibility mask".